nano/node/logging.cpp

```diff
@@ -37,20 +36,6 @@ void nano::logging::init (boost::filesystem::path const & application_path_a)
 		}
 
 #ifdef BOOST_WINDOWS
-		if (nano::event_log_reg_entry_exists () || nano::is_windows_elevated ())
```

Comment:
Does the removal of this code mean that on windows we do not log to file?

Reply:
This means we can't log to the Windows Event Log program, a kind of Windows syslog. I'll raise an issue to reimplement this if we can get the GitHub builders to support the required tool, the Windows Message Compiler mc.ex.. Short of this I don't know anyone that currently uses Windows + Event Log so I wouldn't hold up the boost submodule conversion on this point.

Reply:
Windows is our least supported platform, I do not think we should have extra features on the platform we support the least.
As long as logging to file exists on windows, I don't think we should do anymore than that.